src/data/javascript-quiz.ts

```diff
@@ -1082,6 +1083,17 @@ const javascriptQuiz = [
       "In JavaScript, typeof null is an object.This result of typeof null is actually a bug in the language.",
     Link: "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
   },
+  {
+    Question:
+      "What JavaScript keyword will return the type of a variable when you call it?",
+    Answer: "typeof",
+    Distractor1: "datatype",
+    Distractor2: "this",
+    Distractor3: "var",
+    Explanation:
+      "typeof is a JavaScript keyword that will return the type of a variable when you call it. You can use this to validate function parameters or check if variables are defined. There are other uses as well.",
```

Comment:
I think we could leave off that last part of the question

```suggestion
      "typeof is a JavaScript keyword that will return the type of a variable when you call it. You can use this to validate function parameters or check if variables are defined.",
```